docs/topics/eclipse-configuring-run.adoc

```diff
@@ -20,6 +20,7 @@ You can create multiple run configurations. Each run configuration must have a u
 . On the *Input* tab, complete the following fields:
 .. Select a migration path.
 .. Beside the *Projects* field, click *Add* and select one or more projects.
+// please could i ask for QE input on the following bullet point and warning
 .. Beside the *Packages* field, click *Add* and select one or more the packages.
```

Comment:
packages is removed from UI

Reply:
Thanks - removed in [commit](https://github.com/migtools/mta-documentation/pull/77/commits/c6607b0455dd0997b0e6e150a7839bf23f551740)